common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -476,13 +476,23 @@ trait FeatureSwitches {
   )
 
   // Owner: Simple & Coherent
-  val SupportFrontendActive = Switch(
+  val UkSupportFrontendActive = Switch(
     SwitchGroup.Feature,
     "uk-supporter-traffic-to-new-support-frontend",
     "When ON, all UK membership/contribute/support links send traffic to support.theguardian.com",
     owners = Seq(Owner.withGithub("justinpinner")),
+    safeState = On,
+    sellByDate = new LocalDate(2018, 10, 17),
```

Comment:
This sell by date is yesterday, is that correct?

Reply:
2018

Reply:
Doh! :-)

Reply:
2018

Reply:
whoops, late to the party